draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -119,6 +121,44 @@ public void setClickToMinimizeEnabled(boolean enableClickToMinimize) {
   }
 
   /**
+   * Return if touch listener is enable or disable
+   */
+  public boolean isEnableTouchListener() {
+    return this.enableTouchListener;
+  }
+
+  /**
+   * Enable or disable the touch listener
+   *
+   * @param enableTouchListener to enable or disable the touch event.
+   */
+  public void setEnableTouchListener(boolean enableTouchListener) {
+    this.enableTouchListener = enableTouchListener;
+  }
```

Comment:
This method should use private visibility and not public. You are using it just from DraggableView.